maven/plugins/common/src/main/java/com/vmware/pscoe/maven/plugins/AbstractVroPkgMojo.java

```diff
@@ -52,11 +70,11 @@ private void validateVroPkgParams() {
 			throw new IllegalArgumentException("keystorePassword must be provided.");
 		}
 
-		if (project.getDescription() == null) {
-			this.description = "";
-		} else {
-			this.description = project.getDescription();
+		String descStr = "";
```

Comment:
Can we extract and move this to a separate method? Something like getFormattedDescription?

Reply:
Sure, will do it